storage/migrations/sqlite3/1_init.sql

```diff
@@ -1,7 +1,7 @@
 -- +migrate Up
 
 CREATE TABLE IF NOT EXISTS "boundaries" (
-  "id"                         integer primary key autoincrement,
+  "id"                         integer primary key ,
```

Comment:
Don't remove the autoincrement key this is the correct syntax https://sqlite.org/autoinc.html